### PR TITLE
send reactions

### PR DESCRIPTION
**Pull request by Valdenirmezadri**

### CODE REVIEW

**Comment by mamur-rezeki** [comment]
Reaction is already part of message protocol, doesn't need specific function to be added.

**Comment by tulir** [comment]
`RevokeMessage` is already deprecated so others like that will definitely not be added. `BuildReaction` might make sense, especially if combined with a `BuildMessageKey` to share the code with `BuildRevoke`.

**Comment by wildy283** [comment]
thx

File: send.go [hunk 1]
```diff
@@ -247,6 +247,40 @@ func (cli *Client) BuildRevoke(chat, sender types.JID, id types.MessageID) *waPr
...
 }
 
+// ReactionMessage sends an emoji as a reaction to a message
+//
+// To react your own messages, pass your JID or an empty JID as the second parameter (sender).
+//
+// To react someone else's messages when you are in a group, pass the message sender's JID as the second parameter.
+//
+// To remove a reaction, pass an empty reaction.
+func (cli *Client) ReactionMessage(chat, sender types.JID, id types.MessageID, reaction string) (SendResponse, error) {
```
Comment (tulir) [inline send.go:257]: This method needs to be removed

File: send.go [hunk 2]
```diff
@@ -247,6 +247,40 @@ func (cli *Client) BuildRevoke(chat, sender types.JID, id types.MessageID) *waPr
...
+	return cli.SendMessage(context.TODO(), chat, cli.buildReaction(chat, sender, id, reaction))
+}
+
+func (cli *Client) buildReaction(chat, sender types.JID, id types.MessageID, reaction string) *waProto.Message {
+	key := &waProto.MessageKey{
+		FromMe:    proto.Bool(true),
```
Comment (tulir) [inline send.go:263]: This method should be exported, and a new `BuildMessageKey` should be added to share the code with `BuildRevoke`

**Review by tulir — CHANGES_REQUESTED** [review]

**Pull request closed without merging**